pkg/build/builder/dockerutil.go

```diff
@@ -2,11 +2,21 @@ package builder
 
 import (
 	"os"
+	"time"
 
-	"github.com/fsouza/go-dockerclient"
+	docker "github.com/fsouza/go-dockerclient"
+	"github.com/golang/glog"
 	"github.com/openshift/source-to-image/pkg/tar"
 )
 
+var (
+	// Number of retries of pushing the built Docker image into configured
+	// repository
+	DefaultPushRetryCount = 2
```

Comment:
@smarterclayton @ncdc for now I'm for just doing two attempts, we can make that configurable or put more logic there post-3.0